
default interface name for wifi should be wlan, not eth

---
 src/wl/sys/wl_linux.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -Nurp broadcom-wl-6.30.223.271.orig/src/wl/sys/wl_linux.c broadcom-wl-6.30.223.271/src/wl/sys/wl_linux.c
--- broadcom-wl-6.30.223.271.orig/src/wl/sys/wl_linux.c	2020-01-11 02:09:00.384207338 +0200
+++ broadcom-wl-6.30.223.271/src/wl/sys/wl_linux.c	2020-01-11 02:18:35.306461030 +0200
@@ -221,7 +221,7 @@ module_param(nompc, int, 0);
 #define to_str(s) #s
 #define quote_str(s) to_str(s)
 
-#define BRCM_WLAN_IFNAME eth%d
+#define BRCM_WLAN_IFNAME wlan%d
 
 static char intf_name[IFNAMSIZ] = quote_str(BRCM_WLAN_IFNAME);
 
